src/main/java/org/jvnet/hudson/test/RealJenkinsRule.java

```diff
@@ -608,25 +618,42 @@ private static boolean supportsPortFileName(String war) throws IOException {
 
     private static int readPort(File portFile) throws IOException {
         String s = FileUtils.readFileToString(portFile, StandardCharsets.UTF_8);
+        if (StringUtils.isEmpty(s)) {
```

Comment:
```suggestion
        if (s.isEmpty() {
```
?

Reply:
done in [b9589b3](https://github.com/jenkinsci/jenkins-test-harness/pull/465/commits/b9589b3ad5360095bd830191e7ed3a8073738008)

Reply:
BTW you can use GitHub’s native mechanism for applying suggestions in batch, which has some nice properties:
* sets co-author
* automatically resolves comment